primaryHandler.go

```diff
@@ -206,20 +206,30 @@ func (app *App) handleWebhook(writer http.ResponseWriter, req *http.Request) {
 
 	// verify this is valid from caduceus
 	encodedSecret := req.Header.Get("X-Webpa-Signature")
+	secret, err := base64.StdEncoding.DecodeString(strings.TrimPrefix(encodedSecret, "sha1="))
```

Comment:
I don't know why this worked earlier tonight.  What is sent from caduceus should be a string of hex bytes, **not** base64.

https://github.com/Comcast/caduceus/blob/bd4baa0f7c00ecf1a858de4c5f6db4c2f0bb2d9d/src/caduceus/outboundSender.go#L509

Reply:
yeah, I noticed that right after I pushed. Finally fixed it though. Also figured out that the sig is done on the payload not the message.